sled-agent/src/zone_bundle.rs

```diff
@@ -221,20 +222,12 @@ pub struct ZoneBundler {
     inner: Arc<Mutex<Inner>>,
     // Channel for notifying the cleanup task that it should reevaluate.
     notify_cleanup: Arc<Notify>,
-    // Tokio task handle running the period cleanup operation.
-    cleanup_task: Arc<tokio::task::JoinHandle<()>>,
-}
-
-impl Drop for ZoneBundler {
```

Comment:
Does removing this end up leaking a bunch of cleanup tasks in tests? I wonder if we can make it exit on its own if the `ZoneBundler` that spawned it is dropped? It doesn't look like `Notify` knows if it's the only one left, though. :-/

Reply:
No, it doesn't. All test zpools are destoyed in the [`StorageResources` `Drop` impl](https://github.com/oxidecomputer/omicron/blob/c8db7c88dec4030f29bea0520dea43e81187c8dd/sled-agent/src/zone_bundle.rs#L2260-L2268)